lib/Service/RoomService.php

```diff
@@ -336,6 +336,17 @@ public function setLobby(Room $room, int $newState, ?\DateTime $dateTime, bool $
 		return true;
 	}
 
+	public function setAvatar(Room $room, $avatarName): bool {
+		if ($room->getType() === Room::TYPE_ONE_TO_ONE) {
+			return false;
+		}
+		$update = $this->db->getQueryBuilder();
+		$update->update('talk_rooms')
+			->set('avatar', $avatarName)
+			->where($update->expr()->eq('id', $update->createNamedParameter($room->getId(), IQueryBuilder::PARAM_INT)));
+		return true;
```

Comment:
room object is not modified

Reply:
Needs to dispatch an event, so we can later on create a system message, just compare with all the other methods around this one.